src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -78,6 +78,7 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction Day = _library.Append(new DayFunction());
         public static readonly TexlFunction Degrees = _library.Append(new DegreesFunction());
         public static readonly TexlFunction DegreesT = _library.Append(new DegreesTableFunction());
+        public static readonly TexlFunction Distinct = _library.Append(new DistinctFunction());
```

Comment:
>public static readonly TexlFunction Distinct = _library.Append(new DistinctFunction()); [](http://example.com/codeflow?start=8&length=87)

By adding this function here it means that PA team also need to implement it from their side. Lets check with them first and then decide if we add the function to the _library or _featureGateFunctions collection.

Reply:
Did this get resolved?

Reply:
@anderson-joyle can we add it to _featureGateFunctions for now, at least that unblocks Nathan.

Reply:
Lets get it included to _featureGateFunctions for now.

Reply:
turns out this is already in PA, keeping it under _library

Reply:
My concern is that PAClient has its own implementation of this, [https://msazure.visualstudio.com/DefaultCollection/OneAgile/_git/PowerApps-Client?path=/src/Cloud/DocumentServer.Core/Document/Texl/Builtins/Distinct.cs&version=GBmaster](https://msazure.visualstudio.com/DefaultCollection/OneAgile/_git/PowerApps-Client?path=/src/Cloud/DocumentServer.Core/Document/Texl/Builtins/Distinct.cs&version=GBmaster)

So adding to this list may hide that implementation. 

Let's just move this as a static field on Library.cs, and then it's only used by interpreter.